src/libcore/cell.rs

```diff
@@ -1399,6 +1406,351 @@ impl<T: ?Sized + fmt::Display> fmt::Display for RefMut<'_, T> {
     }
 }
 
+/// A cell which can be written to only once.
+///
+/// Unlike `RefCell`, a `OnceCell` only provides shared `&T` references to its value.
+/// Unlike `Cell`, a `OnceCell` doesn't require `T: Copy` to access its value.
```

Comment:
There's various ways to access the value of a Cell without T being Copy.

Reply:
Ah yes, this statement isn't very accurate so I'll reword it :+1: